### ISSUE TITLE
open served folder in OS's file explorer

### ISSUE THREAD

**Issue by steltenpower** [enjalot/served#6]
Could save some time clicking in the explorer

**Comment by enjalot** [comment]
yeah, we could have a little link that does this. some resources for accomplishing this:
https://discuss.atom.io/t/opening-the-oss-file-explorer/24348/3
